packages/block-library/src/blocks/header/variations.js

```diff
@@ -2,37 +2,127 @@ import * as icons from './icons';
-	},
-	{
-		name: 'logo-center',
-		title: __( 'Logo centered and one navigation menu on each side', '__plugin_txtd' ),
-		icon: icons.logoCenter,
-		innerBlocks:  [
-			[ 'novablocks/navigation', {
-				className: "site-header__menu site-header__menu--secondary",
-				slug: "secondary"
-			} ],
-			[ 'novablocks/logo' ],
-			[ 'novablocks/navigation', {
-				className: "site-header__menu site-header__menu--primary",
-				slug: "primary"
-			} ],
-		],
-		scope: [ 'block' ],
-	}
+  {
+    name: 'logo-left',
+    title: __('Logo left with one menu on the right.', '__plugin_txtd' ),
+    icon: icons.fargo,
+    innerBlocks: [
+      ['novablocks/header-row', {
+        className: 'site-header__row site-header__row--primary',
+        headerRowType: 'primary'
+      },
+        [
+          ['novablocks/logo'],
+          [ 'novablocks/navigation', {
+            className: "site-header__menu site-header__menu--primary",
+            slug: "primary"
+          } ]
+        ]
+      ]
+    ],
+    scope: [ 'block' ],
+  },
+  {
+    name: 'logo-center',
+    title: __('Logo centered with one menu on the left and one menu on the right.', '__plugin_txtd' ),
+    icon: icons.rosa2,
+    innerBlocks: [
+      ['novablocks/header-row', {
+        className: 'site-header__row site-header__row--primary',
+        headerRowType: 'primary'
+      },
+        [
+          [ 'novablocks/navigation', {
+            className: "site-header__menu site-header__menu--secondary",
+            slug: "secondary"
+          } ],
+          ['novablocks/logo'],
+          [ 'novablocks/navigation', {
+            className: "site-header__menu site-header__menu--primary",
+            slug: "primary"
+          } ]
+        ]
+      ]
+    ],
+    scope: [ 'block' ],
+  },
+  {
+    name: 'logo-center-two-rows',
+    title: __('Two Rows Header with centered logo and three menus.', '__plugin_txtd'),
+    icon: icons.heap,
+    innerBlocks: [
+      ['novablocks/header-row', {
+        className: 'site-header__row site-header__row--logo',
+        headerRowType: 'logo'
+      },
+        [
+          [ 'novablocks/navigation', {
+            className: "site-header__menu site-header__menu--secondary",
+            slug: "secondary"
+          } ],
+          ['novablocks/logo'],
+          [ 'novablocks/navigation', {
+            className: "site-header__menu site-header__menu--tertiary",
+            slug: "tertiary"
+          } ]
+        ]
+      ],
+      ['novablocks/header-row', {
+        className: 'site-header__row site-header__row--primary',
+        headerRowType: 'primary'
+      },
+        [
+          [ 'novablocks/navigation', {
+            className: "site-header__menu site-header__menu--primary",
+            slug: "primary"
+          } ]
+        ]
+      ]
+    ],
+    scope: [ 'block' ],
+  },
+  {
+    name: 'logo-center-three-rows',
+    title: __('Three Rows Header with centered logo and three menus.', '__plugin_txtd'),
+    icon: icons.felt,
```

Comment:
`logo-center-three-rows` better explains the intention of this variation than `felt`. Therefore, using theme names for accessing the icons for these variations has the only advantage of being short. This also applies for all the other variations.